docs/satellite-clone.md

```diff
@@ -13,6 +13,12 @@
 #### Prerequisites ####
 
 1. You will need files from a katello-backup (`katello-backup` on the `Satellite server`).
+   Required backup files:
```

Comment:
Do you guys think we should add a note here pointing that "these are the required files from a `Sat6.2` katello-backup. ? On Satellite6.1, katello-backup generates one additional file `elastic_data.tar.gz`